packages/components/card/src/card.tsx

```diff
@@ -44,23 +66,74 @@ const Card = (props: TCardProps) => (
       background: ${props.theme === 'dark'
         ? designTokens.colorNeutral95
         : designTokens.colorSurface};
-    `}
-    // Allow to override the styles by passing a `className` prop.
-    // Custom styles can also be passed using the `css` prop from emotion.
-    // https://emotion.sh/docs/css-prop#style-precedence
-    className={props.className}
-  >
-    {props.insetScale === 'none' ? (
-      // Use a `<div>` to ensure that there is always a wrapper container.
-      // This is mostly useful in case custom styles are targeting this element.
+      cursor: ${props.isDisabled
+        ? 'not-allowed'
+        : isClickable
+        ? 'pointer'
+        : 'default'};
+      :hover {
+        background: ${props.theme === 'dark'
+          ? isClickable
+            ? designTokens.colorNeutral90
+            : undefined
+          : isClickable
+          ? designTokens.colorNeutral98
+          : undefined};
+      }
+      // Disables link text styling
+      color: inherit;
+      // Changes the opacity of the content, not the card itself
+      & > div {
+        opacity: ${props.isDisabled ? 0.5 : 1};
+      }
+    `,
+    className: props.className,
+  };
+
+  const content =
+    props.insetScale === 'none' ? (
       <div>{props.children}</div>
     ) : (
       <Inset scale={props.insetScale} height="expanded">
         {props.children}
       </Inset>
-    )}
-  </div>
-);
+    );
+
+  if (isClickable) {
+    if (props.to) {
+      if (props.isExternalLink) {
+        warning(
+          typeof props.to === 'string',
+          'ui-kit/Card: "to" property must be a string when "isExternal" value is true'
+        );
+
+        return (
+          <a
+            {...commonProps}
+            // @ts-ignore
+            href={props.to}
```

Comment:
Maybe we can use a casting here:
```
href={props.to as string|
```

Reply:
If we have received an object instead of a string, it will be used it's `toString()` under the hood and it will render a string (bad link URL) but it's not the component responsibility and we also rendered a warning.

Reply:
Great thinking. Addressed in 007db829991ecf9c421d3195b7aee93d9bf586ba